test/test_validation.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+var assert = require("assert"),
```

Comment:
Please use single quotes (`'`) for system strings, symbols, identifier-like strings etc. and double quotes (`"`) for natural language strings/messages intended for humans unless they contain double quotes.

I know this is not specified anywhere, so here's an example:
```
var assert = require('assert')

var messages = {
    'hello world': "Hello world!"
}

console.log("What kind of bird are YOU?")

// we can make an exception though
console.log('Never say "never"!')

console.log("You can escape \"s if you REALLY need to, that is the string contains both 's and \"s")
```